src/app/footer/footer.component.ts

```diff
@@ -28,19 +28,19 @@ export class FooterComponent implements OnInit {
 
   formSubmit(email: string) {
     // TODO get notified API path
-    const API_PATH = 'verify-email';
+    const API_PATH = '';
```

Comment:
Ahhhh... @live-wire Not sure if I am late, but One thing I want to point out that we must have a single configuration file for all the API urls rather than defining it in every function. Please let me know if there is some doubt in it.
cc: @deshraj @aka-jain @spyshiv

Reply:
Makes sense! 
I'll do that 👍 
Will create an issue to track it.